planner/core/src/main/java/eu/seaclouds/platform/planner/core/HttpHelper.java

```
@@ -114,11 +141,53 @@ public String postRequest(String restPath, List<NameValuePair> params){
                 response.close();
             } catch (IOException e) {
                 e.printStackTrace();
-                return null;
+                return new Pair<>("500", "Post Exception for: " + httpPost.toString());
+            }
+        }
+    }
+
+    public Pair<String, String> postRequestWithParams(String restPath, List<NameValuePair> params){
+        HttpPost httpPost = new HttpPost(prepareRequestURL(restPath, params));
+        CloseableHttpResponse response = null;
+        try {
+            httpPost.setEntity(new UrlEncodedFormEntity(params));
+            response = httpclient.execute(httpPost);
+            HttpEntity entity = response.getEntity();
+            String content = "";
+            int status = response.getStatusLine().getStatusCode();
+
+            if(status == 200) {
+                content = EntityUtils.toString(entity); //new Scanner(entity.getContent()).useDelimiter("\\Z").next();
+            }else{
+                content =response.getStatusLine().getReasonPhrase();
+            }
+
+            EntityUtils.consume(entity);
+            return new Pair<String, String>(String.valueOf(status), content);
+
+        } catch (UnsupportedEncodingException e) {
+            e.printStackTrace();
+            return null;
+        } catch (ClientProtocolException e) {
+            e.printStackTrace();
+            return null;
```

Use logger and don't ignore exceptions